### PR TITLE
[FAD-68] KYC

### PR DESCRIPTION
**Pull request by sachasmart-weavik**
### Description

> KYC Flow for a user. If a user is does not have a UID they need to complete KYC. 
### Checklist

- [ ] `CHANGELOG` has been updated (_if appropriate_)
- [ ] Environment variables updated (_if appropriate_)

### CODE REVIEW

File: packages/client3/pages/artist/dashboard/index.page.tsx [hunk 1]
```diff
@@ -59,21 +60,22 @@ function DashBoard() {
           onButtonClick={() => {
             //TODO: Dummy function definition below
             if (isVerified) {
+              console.log("verified:", isVerified);
```
Comment (v1n33th) [inline packages/client3/pages/artist/dashboard/index.page.tsx:63]: Are these logs added on purpose?

File: packages/client3/pages/artist/dashboard/index.page.tsx [hunk 2]
```diff
@@ -59,21 +60,22 @@ function DashBoard() {
...
             } else {
+              console.log("nope:", isVerified);
+
               setIsVerified(true);
             }
           }}
         />
       </>
     );
+  } else if (user && !isVerified) {
```
Comment (v1n33th) [inline packages/client3/pages/artist/dashboard/index.page.tsx:74]: `else` not required  as there is return in all conditions

File: packages/client3/services/user-services.ts [hunk 3]
```diff
@@ -9,3 +12,22 @@ export const hasUid = (user?: User) => {
...
   );
 };
+
+export const getKYCStatus = async (
+  account: string,
+  provider: Web3Provider,
+  user?: User
+) => {
+  if (hasUid(user)) {
+    console.log(user);
```
Comment (v1n33th) [inline packages/client3/services/user-services.ts:22]: this log might not be needed

**Review by v1n33th — COMMENTED** [review]
Awesome..

File: packages/client3/pages/artist/dashboard/index.page.tsx (hunk 1 above)
Comment (sachasmart-weavik) [reply packages/client3/pages/artist/dashboard/index.page.tsx:63]: Whoops

**Pull request merged**